beacon_node/beacon_processor/src/lib.rs

```diff
@@ -155,9 +160,15 @@ impl BeaconProcessorQueueLengths {
             aggregate_queue: 4096,
             unknown_block_aggregate_queue: 1024,
             // Capacity for a full slot's worth of attestations if subscribed to all subnets
-            attestation_queue: active_validator_count / slots_per_epoch,
+            attestation_queue: std::cmp::min(
```

Comment:
I may be wrong here, I don't understand why it is `std::cmp::min`. From my understanding, it takes the minimum value of the two in comparison. I thought it should be `std::cmp::max`. 

Because if it is `min`, then when `active_validator_count / slots_per_epoch` is, say 16, then the queue would be still 0? But if we use `max`, then for small validator count, it will return `MIN_QUEUE_LEN` which is 128.

Reply:
you're correct!

Reply:
ups, good catch